dlib/cuda/tensor_tools.h

```diff
@@ -1042,6 +1042,64 @@ namespace dlib { namespace tt
                 - #output.nc() == 1+(data.nc() + 2*padding_x - filters.nc())/stride_x
         !*/
 
+#ifdef DLIB_USE_CUDA
```

Comment:
I thought about it... in CUDA it makes sense because there's a cuDNN function for that, but on the CPU it would do the same thing as calling first the convolution, and then the biases. But maybe you're suggesting that to have a uniform API, right?

Reply:
Right. Just talking about uniformity and simplicity of the api.